react/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,16 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  const wait = ms =>
+    new Promise((resolve, reject) => {
+      if (time === time * time) {
```

Comment:
You are not delaying the function here. Check the hint on the beginning of the file to see how to delay the function.
You can use `setTimeout` to delay the execution of a function and `Date.now()` to calculate how long it actually took to run.